src/Apps/Collect/CollectionApp.tsx

```diff
@@ -83,7 +92,7 @@ export class CollectionApp extends Component<CollectionAppProps> {
 }
 
 export const CollectionAppFragmentContainer = createFragmentContainer(
-  CollectionApp,
+  withSystemContext(CollectionApp),
```

Comment:
`withSystemContext` was crerated to address legacy code; we should be using the hook form

Reply:
It looks like this _is_ a legacy class component. Are you suggesting we convert it to a functional component as part of this work?

Reply:
Ah, nope, since this is already a class all good 👍

Reply:
Can you explain why we add this context here? Is it to pass props to CollectionsHubRails without having to pass through every component in the CollectionsApp tree?

Reply:
@ashleyjelks my understanding is that it gives access to the `user` prop, via `SystemContextProps`.

Reply:
Yeah, this is actually temporary. Sense we aren't ready to launch this yet grabbing the user from the context allows me to conditionally display this to only admins.

Reply:
Got it!